src/components/Marker.js

```diff
@@ -24,16 +32,40 @@ const Marker = ({ coords, imgSrc, tooltip, ...props }) => {
 			Object.keys(options).length > 0 ? options : undefined
 		);
 
+		if (enableCard) {
+			const card = new window.SMap.Card();
+
+			card.getHeader().innerHTML = cardHeaderText;
+			card.getBody().innerHTML = cardBodyText;
+			marker.decorate(window.SMap.Marker.Feature.Card, card);
+		}
+
 		markerLayer?.addMarker(marker);
 
 		return () => {
 			markerLayer?.removeMarker(marker, true);
 		};
-	}, [coords, imgSrc, markerLayer, props, tooltip]);
+	}, [
+		cardHeaderText,
+		cardBodyText,
+		coords,
+		enableCard,
+		imgSrc,
+		markerLayer,
+		props,
+		tooltip,
+	]);
 
 	return null;
 };
 
-Marker.propTypes = markerShape;
+Marker.propTypes = {
+	cardBodyText: string,
+	cardHeaderText: string,
+	coords: shape({ lng: number, lat: number }).isRequired,
+	enableCard: bool,
+	imgSrc: string,
+	tooltip: string,
+};
```

Comment:
Implementation is fine, but we should keep everything in markerShape, and edit MultipleMarkers component with the same functionality (maybe extract Marker creation to separate utils function), or add a TODO with this